structured_experiments/2022.04.28.notebook_template/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3"
+
+services:
+  training_notebook:
+    image: training_notebook
+    container_name: training_notebook

```

Comment:
for future reference, working locally, is this where the container name need to be change if making a copy of the experiment for learning and testing? Perhaps add a comment under "experiment.MD"